widgets/Salesforce-batchsink.json

```diff
@@ -193,6 +193,22 @@
             ],
             "default": "Skip on error"
           }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Data type Validation",
+          "name": "datatypeValidation",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "true"
```

Comment:
Shouldn't the default be false ?

Reply:
it should be true for new pipelines and false for upgrade pipelines. In case of upgrade pipeline, we are setting it as false in isDatatypeValidation method.